voronoi_app.py

```diff
@@ -209,26 +213,25 @@ def reset_svgplot(self, full_refresh=False):
 
     def plot_tiles(self):
         if self.curr_day == 0 and self.curr_state == 0:
-            tmp = [sympy.geometry.Point2D(25, 25), sympy.geometry.Point2D(25, 75),
-                   sympy.geometry.Point2D(75, 75), sympy.geometry.Point2D(75, 25)]
-            q_t = sympy.geometry.Polygon([0, 0], [0, 50], [50, 50], [50, 0])
+            tmp = [Point(25, 25), Point(25, 75),
+                   Point(75, 75), Point(75, 25)]
+            q_t = [[0, 0], [0, 50], [50, 50], [50, 0], [0, 0]]
             for i in range(constants.no_of_players):
-                base_marker = sympy.geometry.Polygon(
-                    *list(map(lambda p: p.translate(tmp[i].x - q_t.centroid.x,
-                                                    tmp[i].y - q_t.centroid.y), q_t.vertices)))
+                off_x = tmp[i].x - 25
```

Comment:
Logic change: inferred that the centroid is 25, 25

Reply:
Is that for any map size?